<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTaskWindow</class>
 <widget class="QDialog" name="addTaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="LE_taskName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TL_taskName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Task name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TE_description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TL_description">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Task description</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="DE_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="DE_end">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_createTask">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
